moveos/moveos-stdlib/moveos-stdlib/sources/entrys.move

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+/// Collection of entry functions
+//TODO maybe need a better name
+module moveos_std::entrys {
+    use std::vector;
+    use moveos_std::context::{Self, Context};
+    use moveos_std::move_module::{Self, MoveModule};
+
+    /// Entry function to publish modules
+    /// The order of modules must be sorted by dependency order.
+    public entry fun publish_modules(ctx: &mut Context, account: &signer, modules: vector<vector<u8>>) {
```

Comment:
Are all framework entry functions defined here?

Reply:
To resolve the dependency issue, `account_storage` can not depend on `context`. Maybe we can put it into the `package.move` module as @pause125 mentioned.